lib/functions/from_list.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class FromList
+  def self.call_params
```

Comment:
sorry, I missed that you were returning a lambda here, so `call_params` is not a good name. Rename to `to_hash`

Reply:
Renamed